makelib/golang.mk

```diff
@@ -284,6 +290,8 @@ Go Targets:
     generate        Runs go code generation.
```

Comment:
do you want to say that this will also run go imports?

Reply:
Yes, that would be good to make that clear from this usage/help text.